models/debian.go

```diff
@@ -87,8 +75,11 @@ func ConvertDebianToModel(root *oval.Root) (defs []Definition) {
 	return
 }
 
-func collectDebianPacks(cri oval.Criteria) []distroPackage {
-	return walkDebian(cri, "", []distroPackage{})
+func collectDebianPacks(cri oval.Criteria) (packs []Package) {
+	for _, distPack := range walkDebian(cri, "", []distroPackage{}) {
+		packs = append(packs, distPack.pack)
+	}
+	return
```

Comment:
When no package is found, I want to return with []Package instead of nil.
```suggestion
func collectDebianPacks(cri oval.Criteria) ([]Package) {
	packs := []Package{}
	for _, distPack := range walkDebian(cri, "", []distroPackage{}) {
		packs = append(packs, distPack.pack)
	}
	return packs
```